lib/commons/aria/index.js

```diff
@@ -473,7 +483,8 @@ lookupTables.role = {
 			one: ['menuitem', 'menuitemradio', 'menuitemcheckbox']
 		},
 		nameFrom: ['author'],
-		context: null
+		context: null,
+		implicit: ['menu[type="context"]']
```

Comment:
Huh, I didn't know you could specify implicit ARIA roles that way (with an attribute selector).